UI/window-basic-settings-stream.cpp

```diff
@@ -482,6 +482,10 @@ void OBSBasicSettings::on_service_currentIndexChanged(int)
 
 	auto system_auth_service = main->auth->service();
 	bool service_check = service == system_auth_service;
+	if (!service_check)
+		service_check = !!main->auth &&
```

Comment:
The null check here is superfluous, there already is one a few lines above with an early return. You could also replace the simple comparison in the line above with the `.find() != ...npos` check.

This change might also need to be reflected in the Auto-Configuration wizard.

Reply:
True.  Fixing it now.  And yes, this does also need changed in auto-config, which I'll also fix.